diagnostic.md

```diff
@@ -48,11 +52,32 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    in the model, you have to pass [params.movie_id -1]
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    if i wanted to access the titles, then:
+
+    {{#each model as |movie|}}
+      <p>{{movie.name}}</p>
+    {{/each}}
+
+    model: function(params){
+    return [
+      {
+        id: 1,
+        name: 'movie1',
+      },
+      {
+        id: 2,
+        name: 'movie2',
+      },
+      {
+        id: 3,
+        name: 'movie3',
+      },
+    ]
+  }
```

Comment:
Great, and to emphasize, the `model` is what provides the data from the Route to the template